<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ARO-MAB-2105" failures="1" tests="1" name="com.automationexercise.tests.TestCase20" time="25.163" errors="0" timestamp="2024-12-03T12:27:09 WAT" skipped="0">
  <testcase classname="com.automationexercise.tests.TestCase20" name="searchProductsAndVerifyCartAfterLogin" time="25.163">
    <failure message="Verify &#039;Cart is empty! Click here to buy products.&#039; is visible expected [Cart is empty! Click here to buy products.] but found [Cart is empty! Click here to buy products.Product catalog]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Verify 'Cart is empty! Click here to buy products.' is visible expected [Cart is empty! Click here to buy products.] but found [Cart is empty! Click here to buy products.Product catalog]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at com.automationexercise.tests.TestCase20.verifyThatCartIsEmpty(TestCase20.java:73)
at com.automationexercise.tests.TestCase20.searchProductsAndVerifyCartAfterLogin(TestCase20.java:48)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
    </failure>
  </testcase> <!-- searchProductsAndVerifyCartAfterLogin -->
  <system-out/>
</testsuite> <!-- com.automationexercise.tests.TestCase20 -->
